usersync/usersyncers/syncer_test.go

```diff
@@ -132,31 +132,3 @@ func TestNewSyncerMap(t *testing.T) {
 		}
 	}
 }
-
-// Bidders may have an ID on the IAB-maintained global vendor list.
```

Comment:
Do we want to move this uniqueness test to `config/bidderinfos_test.go` or just drop it entirely? We can run it against the default configs in the `static/` directory.

Reply:
I removed the test because it's broken and the intent of the test is flawed.

**Broken**
The test uses an empty configuration object which results in empty sync urls which in turn causes the `NewSyncerMap` method to return an empty map. This has been broken for at least 2 years.

**Flawed**
The test looks to ensure each bidder has it's own unique vendor id. However, we have adapters like adkernel/adkernelAdn and triplelift/triplelift_native which legitimately need to have the same vendor id.

I suppose the original intent was to eliminate copy / paste errors, but I don't see we can continue to automate a check like this without allowing for the legitimate cases.

We have also become more thorough in vetting vendor ids during the review process.